### DISCUSSION TITLE
Previous Seasons

### DISCUSSION THREAD

**Discussion by NotLeechD** [AnthonyMichaelTDM/DRG-Save-Editor#65] [Q&A] [answered]
any way to unlock the things from previous seasons? i tried setting to the older season and adding scrip but no luck

**Comment by wahlp** [comment, marked as answer]
i'm working on this in https://github.com/AnthonyMichaelTDM/DRG-Save-Editor/pull/64. would appreciate if you could try it and give feedback there.
